daemon/settings.c

```diff
@@ -865,10 +882,10 @@ appsettings_appname(const appsettings_t *self)
  * ------------------------------------------------------------------------- */
 
 static void
-appsettings_notify_change(appsettings_t *self)
+appsettings_notify_change_ex(appsettings_t *self, bool notify)
```

Comment:
About naming of this: This is notify change method that may skip notifying and do something else instead, which seems a little strange. However I understand that this is "overloading" (in C++ speak) notify change method below. Or the other way around but anyway this seems more like `appsettings_save_later(appsettings_t *self, bool notify)` instead.

Reply:
It is pretty much as you describe. From layering point of view I prefer having only "notification" calls at the level where these happen i.e. variance in how (notify) , not what (notify with implicit save vs. explicit save only).